File: module-wanandroid/src/main/java/com/yyxnb/module_wanandroid/ui/WanActivity.java
package com.yyxnb.module_wanandroid.ui;


import androidx.fragment.app.Fragment;

import com.alibaba.android.arouter.facade.annotation.Route;
import com.yyxnb.common_base.base.ContainerActivity;
import com.yyxnb.common_res.constants.WanRouterPath;

@Route(path = WanRouterPath.MAIN_ACTIVITY)
public class WanActivity extends ContainerActivity {

    @Override
    public Fragment initBaseFragment() {
        return new WanMainFragment();
    }

}

File: module-wanandroid/src/main/java/com/yyxnb/module_wanandroid/ui/WanActivity.kt
package com.yyxnb.module_wanandroid.ui

import androidx.fragment.app.Fragment
import com.alibaba.android.arouter.facade.annotation.Route
import com.yyxnb.common_base.core.ContainerActivity
import com.yyxnb.common_res.arouter.ARouterConstant

@Route(path = ARouterConstant.WAN_MAIN)
class WanActivity : ContainerActivity() {
    override fun initBaseFragment(): Fragment {
        return WanMainFragment()
    }
}